apps/web/pages/welcome.tsx

```diff
@@ -35,6 +39,27 @@ export default function Welcome(props) {
         setStage(stage + 1);
     }
 
+    useEffect(() => {
+        const hrefs = window.location.href.split('/');
+        const len = hrefs[3];
+      
+      switch (len){
+        case "de":
+          messages = de;
+          break;
+        default:
+          messages = en;
+          break;
+      }
+      });
+
```

Comment:
```suggestion

// workaround to prevent lingui from removing translations only used by error messages
```
Might make sense to add a comment explaining the unreachable code here